README.md

```diff
@@ -219,7 +219,7 @@ invokes cmake commands as needed.
 
     ```bash
     cd monero
-    git checkout release-v0.15
+    git checkout release-v0.15.1
```

Comment:
Is that intended ? Up to now it was a x.y branch, further numbers were tags.

Reply:
I thought we will branch `release-v0.15.1` from master. Would you prefer merging master into `release-v0.15`?

Reply:
No. I'd have expected 0.16 then. But then it doesn't really matter, fine that way.